app/src/main/java/com/appunite/loudius/ui/components/LoudiusDialog.kt

```diff
@@ -0,0 +1,72 @@
+package com.appunite.loudius.ui.components
+
+import androidx.compose.material3.AlertDialog
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import com.appunite.loudius.ui.theme.LoudiusTheme
+
+@Composable
+fun LoudiusDialog(
+    onDismissRequest: () -> Unit,
+    confirmButton: @Composable () -> Unit,
+    modifier: Modifier = Modifier,
+    title: String,
+    dismissButton: @Composable (() -> Unit)? = null,
+    /**
+     * For text {@see LoudiusTextStyle#ScreenContent} should be used
```

Comment:
**nitpick:** If you wrap `LoudiusTextStyle` in a [] brackets it will be highlighted by the AS and then you can use the `Go to declaration` shortcut.